python-cache-demo/stats-service/stats_db_conn.py

```diff
@@ -0,0 +1,25 @@
+import mysql.connector
+import logging
+
+class StatisticsDBConnector:
+    def __init__(self, host, user, pwd, database="statistics", logger=logging.Logger(__name__)):
+        self.logger = logger
+        self.logger.info(f"Connecting to the database {database} on the host: {host}.")
+        try:
+            self.db_conn = mysql.connector.connect(
+                host=host,
```

Comment:
same as above, passed as ENVs

Reply:
Ok, i see, the ENVS will be passed in stats_service.py

Reply:
Yes, they are passed when instantiating the class